lib/plugins/Label_H1_FPN.test.ts

```diff
@@ -105,6 +120,45 @@ test('decodes Label H1 Preamble FPN in-flight', () => {
   expect(decodeResult.formatted.items[4].value).toBe('0xddfb');
 });
 
+
+test('decodes Label H1 Preamble with WS', () => {
+  const decoder = new MessageDecoder();
+  const decoderPlugin = new Label_H1_FPN(decoder);
+
+  expect(decoderPlugin.decode).toBeDefined();
+  expect(decoderPlugin.name).toBe('label-h1-fpn');
+  expect(decoderPlugin.qualifiers).toBeDefined();
+  expect(decoderPlugin.qualifiers()).toEqual({
+    labels: ['H1'],
+    preambles: ['FPN'],
+  });
+
+  // https://app.airframes.io/messages/2372685289
+  const text = 'FPN/TS140017,021724/RP:DA:EHAM:AA:KMSP..N55064W000477..N55163W001141..ERAKA..N60000W020000..N61000W030000:WS:N61000W030000,370..N61000W040000..N60000W050000..URTAK:WS:URTAK,380..LAKES:WS:LAKES,400..N57000W070000..N54300W080000..N49000W090000..DLH..COLDD:A:BAINY3:AP:ILS30L(30L)/PR4356,344,360,1060,,,13,,,30,,,P50,M40,36090,,3296,292/DTKMSP,30L,172,215117156D';
```

Comment:
this one is a beast - not sure how to approach it. adding it as a reference

Reply:
I like how you added a comment to the URL of the message for further info if we need it later.

Yeah, this is huge.